airflow/providers/google/marketing_platform/operators/display_video.py

```diff
@@ -247,11 +247,11 @@ def execute(self, context: Dict):
         resource = hook.get_query(query_id=self.report_id)
         # Check if report is ready
         if resource["metadata"]["running"]:
-            raise AirflowException('Report {} is still running'.format(self.report_id))
+            raise AirflowException("Report {} is still running".format(self.report_id))
```

Comment:
f-string maybe?

Reply:
Code is before rebase, sorry. :(
I think it is fixed on master. I will check.

Reply:
Yes, I will fix it.